ui/push-health/Job.jsx

```diff
@@ -19,36 +20,53 @@ class Job extends PureComponent {
     const resultStatus = state === 'completed' ? result : state;
 
     return (
-      <span className="mr-2" key={id}>
-        <a
-          className={`btn job-btn filter-shown btn-sm mt-1 ${getBtnClass(
-            result,
-            failureClassificationId,
-          )} border`}
-          href={getJobsUrl({ selectedJob: job.id, repo, revision })}
-          title={`${jobName} - ${resultStatus}`}
+      <span className="ml-1">
+        <Tooltip
+          tagName="span"
+          useDefaultStyles
+          content={
+            <span className="mr-2" key={id}>
+              <div>{jobName}</div>
+              <div>Result: {resultStatus}</div>
+              {job.result === 'testfailed' && (
+                <div>
+                  Open Log Viewer:
+                  <a
+                    className="logviewer-btn ml-1"
+                    href={getLogViewerUrl(job.id, repo)}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                    title="Open the Log Viewer for this job"
+                  >
+                    <img
+                      style={{ height: '18px' }}
+                      alt="Logviewer"
+                      src={logviewerIcon}
+                      className="logviewer-icon text-light mb-1"
+                    />
+                  </a>
+                </div>
+              )}
+            </span>
+          }
         >
-          {jobSymbol}
-        </a>
-        {failureClassificationId !== 1 && (
-          <FontAwesomeIcon icon={faStar} title="Classified" />
-        )}
-        {job.result === 'testfailed' && (
           <a
-            className="logviewer-btn"
-            href={getLogViewerUrl(job.id, repo)}
-            target="_blank"
-            rel="noopener noreferrer"
-            title="Open the Log Viewer for this job"
+            className={`btn job-btn filter-shown btn-sm mt-1 ${getBtnClass(
```

Comment:
What does the `btn job-btn` do? If this acts like a button, should it be a button element instead? :)

Reply:
Excellent question.  :)  I appreciate that you question little things like this...  So often, as in this, it's a partial copy/paste from Treeherder so that jobs looked about the same.

Some of this styling goes back to some Treeherder history.  We opted to use links for job buttons because it seemed to take up less memory than using actual ``<button>`` objects everywhere.  I think it had to do with event handlers being attached or something along those lines.  

But here, I think I would like these to be links so you can click on them with a ``target=_blank``.  However, I totally neglected to put ``target=_blank`` on them anyway.  Nice going, Cam!  Added now.

But your point is well taken as well in that we don't need to use those old classes here.  So I simplified them.  I should revisit the styling on jobs in Treeherder proper at some point.  Perhaps making them Reactstrap Buttons is now a good idea...

Anyway, thanks for calling this out.  :)

Reply:
I was going to remove jquery (I've assigned the bug to me) but I'm concerned there might be old TH code that is using bootstrap classes that involve interactivity. Would you rather work on that bug at some point? (I'm ok either way, just wondering if it'd be easier for you since you have the historical knowledge).

Reply:
I don't believe I'm using jquery in Treeherder anymore.  I had been using it when selecting next/previous with the ``n`` and ``p`` buttons.  But I moved it to ``querySelectorAll`` so I believe we're in the clear.

But an interesting question about Bootstrap interactivity.  I could imagine we're still using some old Bootstrap classes.  I'd be happy to take that bug.  I seem to recall that I addressed all those items, but it's worth checking.

I'm really fine either way (taking the bug) as well.  I'm sure you'd do a great job, you always do.  :)  But if it feels like something you'd feel more comfortable with me doing, please just need-info me and I'll try to get to it in the next month or so.  I have to put Push Health work first, though.

Reply:
Ok, I'll take a stab at it and have you and sheriffs help test it thoroughly before landing.